src/helpers/query_params.coffee

```diff
@@ -0,0 +1,9 @@
+FormRenderer.queryParams = (value) ->
```

Comment:
FYI: I tried to use https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams instead but it looks like the browser version we're currently targeting does not support it.